conf.toml

```diff
@@ -2,6 +2,16 @@ WebStaticFilesPath = "web"
 
 [AccountManager]
 Type = "unix"
+OAuthRedirectURI = "http://localhost:8080/oauth2callback"
```

Comment:
Assuming this default is for testing only and this will be overridden by most users.

Reply:
Yes.
The "unix" stuff is for running the binary locally during development. I decided to include a real url here instead of empty string to make it easier to figure out what to put as redirect URI in the web client configuration in the OAuth server.